src/firefly/js/ui/HiPSSurveyListDisplay.jsx

```diff
@@ -287,30 +264,52 @@ function fieldReducer(hipsUrl) {
                 tooltip: 'display popular HiPS'
             }};
         }
-        return inFields;
+        return Object.assign({}, inFields);
```

Comment:
You are returning a new Object all the time.  Wouldn't that cause the FieldGroupCntlr to update it state?
I am not sure, but if that's the case, then you should only return a new object when you actually modify it.

Reply:
fixed.